test/api/v3/integration/notifications/prevent-multiple-notification.js

```diff
@@ -0,0 +1,33 @@
+import {
+  createAndPopulateGroup,
+} from '../../../../helpers/api-integration/v3';
+
+describe('Prevent multiple notifications', () => {
+  let partyLeader, partyMembers, party;
+
+  before(async () => {
+    let { group, groupLeader, members } = await createAndPopulateGroup({
+      groupDetails: {
+        type: 'party',
+        privacy: 'private',
+      },
+      members: 4,
+    });
+
+    party = group;
+    partyLeader = groupLeader;
+    partyMembers = members;
+  });
+
+
+  it('does not add the same notification twice', async () => {
+    await partyMembers[0].post(`/groups/${party._id}/chat`, { message: 'Message 1'});
```

Comment:
yeah, multiple messages should be posted concurrently

Reply:
```
 const multipleChatMessages = [];

    multipleChatMessages.push(
      partyMembers[0].post(`/groups/${party._id}/chat`, { message: 'Message 1'}),
      partyMembers[1].post(`/groups/${party._id}/chat`, { message: 'Message 2'}),
      partyMembers[2].post(`/groups/${party._id}/chat`, { message: 'Message 3'}),
      partyMembers[3].post(`/groups/${party._id}/chat`, { message: 'Message 4'})
    );

    await Promise.all(multipleChatMessages);

```

won't change the result, its still showing only one notification 

Any other idea to create a test for this?

Reply:
hmm no idea, sorry, I think it might only be reproducible in production where there's more latency